src/server/rest/v1/service/TagService.ts

```diff
@@ -547,6 +712,51 @@ export default class TagService {
     return result;
   }
 
+  private static async unassignTags(tenant: Tenant, action: ServerAction, loggedUser: UserToken, visualTagsIDs: string[]): Promise<ActionsResponse> {
+    const result: ActionsResponse = {
+      inSuccess: 0,
+      inError: 0
+    };
+    // Delete Tags
+    for (const visualTagsID of visualTagsIDs) {
+      try {
+        let tag = await TagStorage.getTagByVisualID(tenant, visualTagsID);
```

Comment:
Remove get of tag as it is already done in 725 with method checkAndGetTagAuthorization()